sfge/src/test/java/com/salesforce/graph/ops/expander/ApexPathExpanderTest.java

```diff
@@ -151,6 +151,65 @@ public void testRegistryCleanup() {
         MatcherAssert.assertThat(path.getInvocableVertexToPaths().entrySet(), hasSize(equalTo(3)));
     }
 
+    @Test
+    public void testLazyExpansion() {
```

Comment:
I verified experimentally that this test fails without my changes, and passes with them.